source/docs/flow_control.md

```diff
@@ -280,7 +283,7 @@ The high watermark path is as follows:
    `Network::ConnectionCallbacks::onAboveWriteBufferHighWatermark()`.
  * When `Envoy::Http::CodecClient` receives `onAboveWriteBufferHighWatermark()` it
    calls `onUnderlyingConnectionAboveWriteBufferHighWatermark()` on `codec_`.
- * When `Envoy::Http::ConnectionManagerImpl` receives `onAboveWriteBufferHighWatermark()` it calls
+ * When `Envoy::Http::Http2::ConnectionImpl` receives `onAboveWriteBufferHighWatermark()` it calls
```

Comment:
Thanks for catching this. Minor adjustment as both H1 and H2 connections will received `onUnderlyingConnectionAboveWriteBufferHighWaterMark` before further propagating.